code/modules/admin/admin_verbs.dm

```diff
@@ -1299,7 +1300,7 @@
 
 /client/proc/get_togglebuildmode()
 	set name = "Toggle Build Mode"
-	set category = "Fun"
+	set category = "Admin.Fun"
```

Comment:
Random unrelated category moves. Should ask admins if they care about their verbs being moved around.